app/bundles/CoreBundle/Tests/Unit/Command/CleanupMaintenanceCommandTest.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace Mautic\CoreBundle\Tests\Unit\Command;
+
+use Mautic\CoreBundle\Command\CleanupMaintenanceCommand;
+use Mautic\CoreBundle\CoreEvents;
+use Mautic\CoreBundle\Event\MaintenanceEvent;
+use Mautic\CoreBundle\Helper\CoreParametersHelper;
+use Mautic\CoreBundle\Helper\IpLookupHelper;
+use Mautic\CoreBundle\Helper\PathsHelper;
+use Mautic\CoreBundle\Model\AuditLogModel;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class CleanupMaintenanceCommandTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @var AuditLogModel|mixed|\PHPUnit\Framework\MockObject\MockObject
+     */
+    private $auditLogModel;
+
+    /**
+     * @var IpLookupHelper|mixed|\PHPUnit\Framework\MockObject\MockObject
+     */
+    private $ipLookupHelper;
+
+    /**
+     * @var PathsHelper|(PathsHelper&object&\PHPUnit\Framework\MockObject\MockObject)|(PathsHelper&\PHPUnit\Framework\MockObject\MockObject)|(object&\PHPUnit\Framework\MockObject\MockObject)|\PHPUnit\Framework\MockObject\MockObject
```

Comment:
This looks a bit excessive. Isn't this enough?
```suggestion
     * @var PathsHelper&\PHPUnit\Framework\MockObject\MockObject
```

But I'd suggest to use the real services instead of the mock and set the `cache_path` param in the `setUp` method to what you need. This could be a simple functional test extending `MauticMysqlTestCase`. It will test all the real services together so we'll really know if it works. You can then remove the whole `getCommandTester` method and simplify the test considerably.

Example:
```php
public function setUp(): void
    {
        $this->configParams['cache_path']  = __DIR__.'/resource/cache/tmp';

        parent::setUp();
    }
```

Reply:
Thanks, I've removed it and add check to audit log after maintenance tests command.